app/views/shared/views/ems_common/_show.html.haml

```diff
@@ -12,5 +12,7 @@
       var miq_after_onload = "miqAsyncAjax('#{url_for(:action => @ajax_action, :id => @record)}');"
   - elsif @showtype == "dialog_provision"
     = render :partial => "shared/dialogs/dialog_provision"
+  - elsif @display == "main"
+    = render :partial => @showtype
```

Comment:
@simon3z to show timeline you should set @showtype = "timeline" in your changes then it should fall thru https://github.com/ManageIQ/manageiq/blob/master/app/views/shared/views/ems_common/_show.html.haml#L9 and render correct partial.